packages/medusa/src/loaders/database.ts

```diff
@@ -54,12 +52,15 @@ export default async ({
       (configModule.projectConfig.database_logging || false),
   } as DataSourceOptions)
 
-  await dataSource.initialize()
-
-  if (isSqlite) {
-    await dataSource.query(`PRAGMA foreign_keys = OFF`)
-    await dataSource.synchronize()
-    await dataSource.query(`PRAGMA foreign_keys = ON`)
+  try {
+    await dataSource.initialize()
+  } catch (err) {
+    // database name does not exist
+    if (err.code === "3D000") {
```

Comment:
comment: Hopefully, that'll reduce friction for developer getting quickly started.

<img width="1199" alt="CleanShot 2023-05-07 at 15 05 09@2x" src="https://user-images.githubusercontent.com/59018053/236679377-b9455c29-eafd-40c1-9169-aba126e2e167.png">

Reply:
question: Should we always throw an error to prevent swallowing it?